pkg/be_controller/be_pod.go

```diff
@@ -177,8 +177,7 @@ func (be *BeController) buildPodTemplate(src *v1alpha12.StarRocksCluster, beconf
 			}}},
 		},
 		LivenessProbe: &corev1.Probe{
-			FailureThreshold: 60,
```

Comment:
I understand there is a default value for `FailureThreshold` .
however, would it be better to set it explicitly for better readability?

Reply:
OK